packages/vizbuilder/src/helpers/chartconfig.js

```diff
@@ -0,0 +1,334 @@
+import {
+  BarChart,
+  Donut,
+  Geomap,
+  LinePlot,
+  Pie,
+  StackedArea,
+  Treemap
+} from "d3plus-react";
+import {sortByCustomKey} from "./sorting";
+
+export const charts = {
+  barchart: BarChart,
+  barchartyear: BarChart,
+  donut: Donut,
+  geomap: Geomap,
+  // histogram: BarChart,
+  lineplot: LinePlot,
+  pie: Pie,
+  stacked: StackedArea,
+  treemap: Treemap
+};
+
+export const ALL_YEARS = "All years";
+
+export const tooltipGenerator = (label, drilldowns, measure, moe) => {
+  const tbody = Array.from(drilldowns)
+    .filter(d => d !== label)
+    .map(dd => [dd, d => d[dd]]);
+  tbody.push([measure, d => `${(d[measure] * 1 || 0).toFixed(4)}`]);
+  if (moe) {
+    moe = moe.name;
+    tbody.push(["Margin of Error", d => `±${(d[moe] * 1 || 0).toFixed(4)}`]);
```

Comment:
For formatting these tooltip values, use `formatAbbreviate` from `d3plus-format`

Reply:
applied in 48bb01690be619a1e59f2e1c72a48707ddd723fb